composer.json

```diff
@@ -21,21 +21,21 @@
         "rss": "https://github.com/laminas/laminas-servicemanager/releases.atom"
     },
     "require": {
-        "php": "~8.1.0 || ~8.2.0 || ~8.3.0",
-        "laminas/laminas-stdlib": "^3.17",
+        "php": "~8.1.0 || ~8.2.0 || ~8.3.0 || ~8.4.0",
+        "laminas/laminas-stdlib": "^3.19",
         "psr/container": "^1.0"
     },
     "require-dev": {
         "composer/package-versions-deprecated": "^1.11.99.5",
-        "friendsofphp/proxy-manager-lts": "^1.0.14",
-        "laminas/laminas-code": "^4.10.0",
+        "friendsofphp/proxy-manager-lts": "^1.0.18",
+        "laminas/laminas-code": "^4.14.0",
```

Comment:
Should a version that supports 8.4 also be added here? ([4.15.0](https://github.com/laminas/laminas-code/releases/tag/4.15.0))

Bumps stdlib to an 8.4 compatible release

Reply:
If it is not necessary, then it can stay that way.